### ISSUE TITLE
Change user password windows not accessible

### ISSUE THREAD

**Issue by monfero** [odoo/odoo#38541]
Impacted versions:
[12]
Steps to reproduce:
After creating a new user by using nvda reader (for vision impaired people) and then try to change password throw menus.
Current behavior:
Change user password window is not accessible and nvda reader doesn't say anything.
Expected behavior:
Other text fields have their own tag so nvda can read them but this fields have no tags.
Video/Screenshot link (optional):
![imaxe](https://user-images.githubusercontent.com/1155130/66676817-a3c71d00-ec68-11e9-9644-5ad83b645433.png)

**Comment by pedrobaeza** [comment]
@mart-e can this have an `aria-label`?

**Comment by mart-e** [comment]
@monfero thanks, if you have suggestions for us to improve the accessibility, we would gladly accept it.

@pedrobaeza sure but this is a list view, where should we add it?

**Comment by pedrobaeza** [comment]
Is not possible to put it on fields on tree views?

**Comment by C3POdoo** [comment]
Dear @monfero,

Thank you for your report but we are closing it due to inactivity.
We apology if we could not look at your request in time.
If your report still makes sense, don't hesitate to reopen a new one. We will try to check it as soon as possible.

_This is an automated message._
